feat: Update Dependencies FAQ page DOCS-607 (#1897)
codacy/docs: docs/faq/code-analysis/does-codacy-check-for-dependencies.md

# Does Codacy check for dependencies?

> Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as issues in your Codacy dashboard.

Reviewer: The **Codacy dashboard** is the Security monitor, right? If it is, we should use the name of the product feature, and we can also add a link to its page. WDYT?

```suggestion
Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as issues in the [Security monitor](../../repositories/security-monitor.md).
```

Author: Not necessarily only the Security monitor. Since these are generic Codacy Issues, they are displayed everywhere a normal issue is displayed: in the Issues tab, in the Security monitor if it's a security issue. In fact, now that you point this out, it may be simpler to call them Issues and not be too specific. Thoughts?

```suggestion
Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as Codacy issues.
```

Reviewer: Yes, makes sense!

> Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as issues in the [Security monitor](../../repositories/security-monitor.md).

Reviewer 2: do we want to promote the Security Monitor if we're thinking about removing it? Could we say the Issues page instead? 🤔

Reviewer 3: indeed the Security Monitor (until we migrate it) is the only place where you can filter by all vulnerable dependencies (InsecureModuleLibraries category)

Author: Agreed. Let's simply call them "Codacy issues".

```suggestion
Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as Codacy issues.
```

> For a list of supported languages and manifest files, see Codacy's [supported languages and tools](../../getting-started/supported-languages-and-tools.md).

Reviewer: For a better connection between both content, I would try to fit the column title **Dependency vulnerability scanning** on this sentence...  

Would this work? 🤔
Is it important to mention the manifest files?

```suggestion
For a list of supported languages and dependency vulnerability scanning tools and manifest files, see Codacy's [supported languages and tools](../../getting-started/supported-languages-and-tools.md).
```

Author: Good point. I'd just reorder the sentence, since "language" and "manifest file" are both objects of the action (what's scanned) and "scanning tools" is what does the scanning. I'm not a fan of those stacked modifiers before tools ("dependency vulnerability scanning tools" is a bit hard to read), but it makes sense for the users who follow the link.

```suggestion
For a list of supported languages and manifest files scanned by the Codacy dependency vulnerability scanning tools, see [Supported languages and tools](../../getting-started/supported-languages-and-tools.md).
```

Reviewer: Looks good!
